src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SubqueryIT.java

```diff
@@ -0,0 +1,138 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Locale;
+
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestsConstants.TEST_INDEX_ACCOUNT;
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestsConstants.TEST_INDEX_DOGSUBQUERY;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.hitAll;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.kvInt;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.kvString;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.core.Is.is;
+
+public class SubqueryIT extends SQLIntegTestCase {
+
+    @Override
+    protected void init() throws Exception {
+        loadIndex(Index.ACCOUNT);
+        loadIndex(Index.DOGSSUBQUERY);
+    }
+
+    @Test
+    public void test() throws IOException {
+        String query = String.format("SELECT * FROM %s/dog A JOIN %s/account B ON A.v = B.v",
```

Comment:
Those format strings are unnecessary.

1) you need to use Locale there
2) I recommend to inline all index names, and make this query single string. It will help later in understanding what's going on (you can just copy query to kibana) and will give better overview of what's happening

(here and everywhere).

I'm open to conversation about it, but I find the following style of tests

```
        JSONObject aggregations = executeQuery(
                "SELECT date_format(insert_time, 'dd-MM-YYYY') " +
                        "FROM elasticsearch-sql_test_index_online " +
                        "GROUP BY date_format(insert_time, 'dd-MM-YYYY') " +
                        "ORDER BY date_format(insert_time, 'dd-MM-YYYY') DESC")
                .getJSONObject("aggregations");

```

much more readable.

Here and everywhere

Reply:
Done. reformat the query.